src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1398,6 +1398,7 @@ public static void setCrossVpcInternodeCommunication(boolean setting)
         conf.cross_vpc_internode_communication_enabled = setting;
     }
 
+    @Deprecated
```

Comment:
Let's file a github issue to remove this entirely at some point?

Reply:
done https://github.com/palantir/cassandra/issues/456